pilot/pkg/xds/sds.go

```diff
@@ -18,6 +18,7 @@ import (
 	"crypto/x509"
 	"encoding/pem"
 	"fmt"
+	"istio.io/istio/pilot/pkg/credentials/kube"
```

Comment:
please run `make gen & make lint`

Reply:
Thanks, addressed the style issue. but I got such an error to run `make gen`.

```
bin/update_crds.sh
Cloning into '/tmp/tmp.FlyhketY9O'...
The authenticity of host 'github.com (140.82.114.3)' can't be established.
ED25519 key fingerprint is SHA256:+DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU.
This key is not known by any other names
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Could not create directory '/home/user/.ssh' (No such file or directory).
Failed to add the host to the list of known hosts (/home/user/.ssh/known_hosts).
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```

Seems it's something unexpected in the base docker image, will take a deep look later.